src/routers/createConfigGetter.js

```diff
@@ -98,7 +97,5 @@ export default (
     );
     outputConfig = applyConfig(routeScreenConfig, outputConfig, configOptions);
 
-    validateScreenOptions(outputConfig, route);
```

Comment:
any reason to kill this? Seems like it provides useful feedback for people who aren't configuring things properly